.github/workflows/dev-deploy.yml

```
@@ -0,0 +1,111 @@
+        EC2_USER: ${{ secrets.EC2_USERNAME }}
+      run: |
+        scp -o StrictHostKeyChecking=no -i ec2_key.pem docker-compose.yml ${EC2_USER}@${EC2_HOST}:~/
+        scp -o StrictHostKeyChecking=no -i ec2_key.pem docker-compose.override.yml ${EC2_USER}@${EC2_HOST}:~/
+
+
+    - name: pull docker and run server
+      env:
+        EC2_HOST: ${{ secrets.EC2_HOST }}
+        EC2_USER: ${{ secrets.EC2_USERNAME }}
+        EC2_SSH_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:48: Double quote to prevent globbing and word splitting

(shellcheck)